From ed8a36d18e7a07f1cc8725f62ec3cc977cfc55cf Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Wed, 14 Jun 2023 16:46:19 +0200
Subject: [PATCH 0377/1141] arm64: dts: st: add memory-region in hpdma nodes on
 stm32mp257f-ev1

Linked-List Items (LLI) can be allocated in internal ram defined under
reserved-memory node.

Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Change-Id: I5afb22351089bad1702116736df3253df3eb93e8
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/317420
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index bcda39709c42..625203cf8697 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -277,6 +277,18 @@ phy1_eth2: ethernet-phy@1 {
 	};
 };
 
+&hpdma {
+	memory-region = <&hpdma1_lli>;
+};
+
+&hpdma2 {
+	memory-region = <&hpdma2_lli>;
+};
+
+&hpdma3 {
+	memory-region = <&hpdma3_lli>;
+};
+
 &i2c2 {
 	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&i2c2_pins_a>;
-- 
2.39.2

